BimIfcExplorer.py

```diff
@@ -223,14 +223,18 @@ def insert(self):
         "inserts selected objects in the active document"
         
         import importIFC
+        from PySide import QtCore,QtGui
         doc = FreeCAD.ActiveDocument
         if doc and self.filename:
             item = self.tree.currentItem()
             if item:
                 eid = item.data(0,QtCore.Qt.UserRole)
                 if eid:
                     importIFC.ZOOMOUT = False
-                    importIFC.insert(self.filename,doc.Name,only=[eid])
+                    try:
+                        importIFC.insert(self.ifc,doc.Name,only=[eid])
```

Comment:
This line is likely not to run, importIFC.insert() is not made to handle self.ifc directly... Are you sure this is what you intended?

Reply:
Yes, this is exactly why it is wrapped in 'try'. There is https://github.com/FreeCAD/FreeCAD/pull/3630, which makes this code work. And the wonders of exception handling allow us to stay backwards compatible.

I believe I tried all possible combinations and it works flawlessly.

Reply:
Oh sorry :) seeing the PR just now. Perfect then!